omod/src/main/resources/config.xml

```diff
@@ -17,7 +17,10 @@
 	<!-- Change the require version back to the below, and update the openMRSVersion property accordingly when 1.6.6 is released -->
 	<!-- require_version>${openMRSVersion}</require_version -->
 	<!-- / Module Properties -->
-
+	<require_modules>
+        <require_module version="3.9">org.openmrs.module.uiframework</require_module>
+    </require_modules>
+	
```

Comment:
Do we still need this if you're importing owasp encoders directly?

Reply:
Fixed